roles/rsyslog/handlers/main.yml

```diff
@@ -3,3 +3,10 @@
   service:
     name: rsyslog
     state: restarted
+  when: not __rsyslogd_restarted | d(false)
+  listen: 'restart rsyslogd'
+
+- name: set rsyslogd restarted
+  set_fact:
+    __rsyslogd_restarted: true
```

Comment:
how will this work if the role gets called multiple times in the same playbook?

Reply:
> how will this work if the role gets called multiple times in the same playbook?

I don't think it will work, so we will need a workaround for the workaround e.g. reset `__rsyslogd_restarted: false` somehow